### PR TITLE
Use running configuration for sonic ports

### PR DESCRIPTION
**Pull request by iljarotar**
## References

Replaces https://github.com/metal-stack/metal-roles/pull/323

## Additional Description

This is a compromise that doesn't introduce any breaking changes. Default mtu, speed and fec for SONiC ports can still be used. Additionally, if the running configuration holds values for mtu, speed and fec these values will be considered for the new config. The reasoning behind this PR is that all values that are not explicitly defined should remain as they are, while the current template will remove mtu and fec from ports, that are not part of the `sonic_ports` dictionary.

### CODE REVIEW

File: partition/roles/sonic/templates/metal.yaml.j2 [hunk 1]
```diff
@@ -107,15 +107,23 @@ PORT:
     {% if 'parent_port' in running_cfg %}
     parent_port: {{ running_cfg.parent_port }}
     {% endif %}
+    admin_status: up
     {% if sonic_ports_dict[name] is defined %}
     {% set port = sonic_ports_dict[name] %}
-    admin_status: up
```
Comment (robertvolkmann) [inline partition/roles/sonic/templates/metal.yaml.j2:112]: Why was this changed?

Comment (iljarotar) [reply partition/roles/sonic/templates/metal.yaml.j2:112]: To make deployments less verbose. Otherwise you have to explicitly add every port to the `sonic_ports_dict`, where you want to have `admin_status: up`.

Comment (robertvolkmann) [reply partition/roles/sonic/templates/metal.yaml.j2:112]: Could you allow to deactivate a port?

**Review by robertvolkmann — APPROVED** [review]
